source/includes/common-mc-admin-config.rst

```diff
@@ -1122,4 +1122,143 @@ MinIO returns an error if the Kakfa broker verison does not match those specifie
 
 A comment to associate with the configuration.
 
-.. end-minio-kafka-audit-logging-comment-desc
\ No newline at end of file
+.. end-minio-kafka-audit-logging-comment-desc
+
+.. start-minio-data-compression-allow_encryption-desc
+
+Set to ``on`` to encrypt objects after compressing them.
+Defaults to ``off``.
+
+.. admonition:: Encrypting compressed objects may compromise security
+   :class: warning
+
+   MinIO strongly recommends against encrypting compressed objects.
+   If you require encryption, carefully evaluate the risk of potentially leaking information about the contents of encrypted objects.
+	   
+.. end-minio-data-compression-allow_encryption-desc
+
+.. start-minio-data-compression-comment-desc
+
+Specify a comment to associate with the data compression configuration.
+   
+.. end-minio-data-compression-comment-desc
+
+.. start-minio-data-compression-enable-desc
+
+Set to ``on`` to enable data compression for new objects.
+Defaults to ``off``.
+
+Enabling or disabling data compression does not change existing objects.
+
+.. end-minio-data-compression-enable-desc
+
+.. start-minio-data-compression-extensions-desc
+
+Comma-separated list of the file extensions to compress.
+Setting a new list of file extensions replaces the previously configured list.
+Defaults to ``.txt,.log,.csv,.json,.tar,.xml,.bin``.
+
+.. admonition:: Incompressable files
+   :class: note
+
+   MinIO will *not* compress the following types of files, even if specified in an :mc-conf:`~compression.extensions` argument:
+
+   .. list-table:: Incompressable File Extensions
```

Comment:
Same issue about how best to format the default settings. When we have a reasonable solution, I'll apply it to all the locations that discuss defaults.